keyserver/README.md

```diff
@@ -36,7 +36,9 @@ Use "coniksserver [command] --help" for more information about a command.
 is for the registration proxy, the second is the server's public address
 for "read-only" requests (lookups, monitoring etc).
 - Edit the configuration file as needed:
-    - Replace the `epoch_deadline` with the desired duration in seconds.
+    - Replace the `database` with the directory location of the persistent storage. However, this value is currently unused.
```

Comment:
> It just seems to be odd to me when we have WithDB in each test (which I don't really want to remove it for now :( ).

Makes sense. To a developer reading our code though, it might just be confusing to have some "random" db-related code.